monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -233,7 +233,7 @@ groups:
             rate of the past 48 hours.
 
       - alert: MTU Mismatch
-        expr: node_network_mtu_bytes{device!="lo"} != on() group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}))
+        expr: node_network_mtu_bytes{device!="lo"} * (node_network_up{device!="lo"} > 0) != on() group_left() (quantile(0.5, node_network_mtu_bytes{device!="lo"}))
```

Comment:
Based on the comments we got from this issue, I think we should additionally ignore the MTU value of interfaces down... and for that we'll need the NIC metadata table!! 🙈